src/styles/blocks/card.scss

```diff
@@ -0,0 +1,139 @@
+.card {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: $main-card-width + 100px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  padding-top: 32px;
+
+  font-family: 'Roboto ', sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+
+  transition: $main-transition;
+
+  &__img {
+    width: $width-img;
+    height: $hight-img;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    width: $width-content;
+    font-size: 1.2em;
+    font-weight: 500;
+    line-height: 18px;
+    color: rgba(6, 11, 53, 1);
+
+    margin: 0 0 4px;
+
+    transition: $main-transition;
+  }
+
+  &__description {
+    width: $width-content;
+    font-weight: 400;
+    font-size: 1em;
```

Comment:
As with the previous comment on font-size, ensure consistency in units across the project. If 'em' units are required by the project's design specifications, disregard this comment.